orbstation/xcard/xenotoxin.dm

```diff
@@ -0,0 +1,113 @@
+// values concerning stamina loss
+#define STAM_DAMAGE_MIN 1
+#define STAM_DAMAGE_MAX 5
+// note that this is a percent, not a value from 0 to 1
+#define STAM_DAMAGE_RATE 15
+
+// values concerning the occasional burst of damage
+// how much more damage a damage burst frame does than a normal one. this effectively goes up by one every damage burst
+#define DAMAGE_BURST_MUL 7
+// again, note that this is a percent chance
+#define DAMAGE_BURST_RATE 5
+#define JITTER_TIME 10 SECONDS
+// note that unlike tox damage, stamina damage, etc, organ damage is NOT multiplied by DAMAGE_BURST_MUL or increased by num_damage_bursts
+#define ORGAN_DAMAGE_MIN 3
+#define ORGAN_DAMAGE_MAX 10
+
+// initial chance of a damage burst being "bad" (dealing extra damage, causing jitters, and displaying a scarier warning message)
+#define BAD_BURST_CHANCE_MIN 10
+// the health value below which having lower health starts to increase the chance of having a bad damage burst
+#define BAD_BURST_HEALTH_RAMP_THRESHOLD 50
+// the percentage above which having low health no longer increases the probability of a bad damage burst
+#define BAD_BURST_CHANCE_SOFTCAP 60
+// chance to scream on a bad damage burst
+#define SCREAM_PROB 50
+
+/datum/reagent/toxin/xenotoxin
+    name = "Xenotoxin"
+    description = "A powerful venom injected by immature xenomorphs when they fail infect a host. \
+                    Lingers in the system for a long time, causing the victim to continuously weaken."
+    color = "#1a011c" // not sure the best color, so i went with xeno-ish purple/black
+    toxpwr = 0.25 // deals low damage but spends a long, long time in your system
+    ph = 1
+    taste_description = "acrid bile"
+    penetrates_skin = TOUCH
+    metabolization_rate = 0.1 * REAGENTS_METABOLISM
+
+    // number of times a damage burst has been triggered
+    // the more bursts you suffer, the more damage each one does and the higher the chance that you'll get "bad bursts" that deal extra damage
+    var/num_damage_bursts = 0
+
+/datum/reagent/toxin/xenotoxin/on_mob_life(mob/living/carbon/victim, delta_time, times_fired)
+
```

Comment:
There's a couple of places here with newlines following the proc declaration, you probably don't need those

Reply:
Keeping these for now, as I think they increase readability. If somebody else wants me to remove 'em as well, I will.